mirth.rb

```diff
@@ -21,15 +21,15 @@ def index
 
   def create
     DailyData.create(date: params['date'], step_count: params["step_count"], notes: params["notes"])
-    redirect_to("/daily_data", status: 303)
+    redirect_to(@@router.url_helpers.daily_data_path, status: 303)
   end
 end
 
-router = ActionDispatch::Routing::RouteSet.new
+@@router = ActionDispatch::Routing::RouteSet.new
```

Comment:
I'm not sure if this is the solution you were hinting at. So I'm also not sure using class variables is the right solution. 😅  We can also use global variables? But I guess the goal is to share the `router` object? 

If this were a Rails app we would have access to `Rails.application.routes`

Reply:
A class variable is a bit surprising; a global variable (or a constant) would be a better way. But an _even_ better way is to (from outside the controller class, when `router` is in scope) do `DailyDataController.include router.url_helpers`, and then those helper methods are just available inside the controller and the template.

Reply:
(The reason `Rails` is available everywhere is because it’s a constant — it begins with a capital letter — so there’s nothing stopping you doing the same here with `Router`. But including the module in the controller is neater, I think, because then the controller itself doesn’t need to know where they come from, it’s just another bit of “wiring up” that you have to do from the outside, like connecting to the database or configuring the view path.)